### PR TITLE
FEAT/add-copy-button-to-copy-code

### PR DESCRIPTION
**Pull request by Tunki1201**
FEAT/add-copy-button-to-copy-code of module page

### CODE REVIEW

File: shared/components/code-box/index.tsx [hunk 1]
```diff
@@ -44,12 +46,24 @@ const CodeBox = ({
...
 
+  const handleCopy = () => {
+    const sourceCode = sampleCodes[selectedLanguage].sourceCode;
+    navigator.clipboard.writeText(sourceCode).then(() => {
+      setCopied(true);
+      setTimeout(() => setCopied(false), 2000);
+    });
+    toast.toast({ title: 'Copied to clipboard' });
+  };
+
```
Comment (gilbertoaceville) [inline shared/components/code-box/index.tsx:66]: Create a custom hook called `useCopy`, this code will be reuseable in the near future.

**Review by gilbertoaceville — COMMENTED** [review]
Make the necessary changes and merge.

**Review by gilbertoaceville — CHANGES_REQUESTED** [review]
Make the necessary changes on `shared/components/code-box/index.tsx`(a comment was left there) and merge.

File: shared/components/code-box/index.tsx (hunk 1 above)
Comment (Tunki1201) [reply shared/components/code-box/index.tsx:66]: ok

Comment (Tunki1201) [reply shared/components/code-box/index.tsx:66]: thank you for your review. I think the copy feature is needed on code box

**Review by gilbertoaceville — APPROVED** [review]

**Pull request merged**